ws-service/types.ts

```diff
@@ -17,6 +17,7 @@ export interface SimplifiedTransaction {
   timestamp: number
   address: string
   rawMessage: string
+  inputAddresses?: Array<string | undefined>
```

Comment:
I don't get this typing (here and in the other parts of this PR where this same type appears): `Array<string | undefined>` . The function you created, `getInputAddresses`, returns `string[]`, so why this undefined?